x-pack/plugins/cases/server/common/models/commentable_case.ts

```diff
@@ -119,6 +119,11 @@ export class CommentableCase {
     return this.subCase?.id;
   }
 
+  private get owner(): string {
+    // TODO: check for subCase?.attributes.owner here
```

Comment:
intentionally left for todo?

Reply:
Good point, I'll add it to the list of sub cases work and remove the TODO.

https://github.com/elastic/kibana/issues/94115